src/concurrently.spec.ts

```diff
@@ -253,6 +253,62 @@ it('uses overridden cwd option for each command if specified', () => {
     );
 });
 
+it('uses raw from options for each command', () => {
+    create(
+        [
+            { command: 'echo', env: { foo: 'bar' } },
+            { command: 'echo', env: { foo: 'baz' } },
+            'kill',
+        ],
+        {
+            raw: true,
+        }
+    );
+
+    expect(spawn).toHaveBeenCalledTimes(3);
+    expect(spawn).toHaveBeenCalledWith(
+        'echo',
+        expect.objectContaining({
+            env: expect.objectContaining({ foo: 'bar' }),
+            stdio: 'inherit',
+        })
+    );
+    expect(spawn).toHaveBeenCalledWith(
+        'echo',
+        expect.objectContaining({
+            env: expect.objectContaining({ foo: 'baz' }),
+            stdio: 'inherit',
+        })
+    );
+    expect(spawn).toHaveBeenCalledWith(
+        'kill',
+        expect.objectContaining({
+            env: expect.not.objectContaining({ foo: expect.anything() }),
+            stdio: 'inherit',
+        })
+    );
+});
```

Comment:
Can we simplify this a bit? E.g. just two commands. Right now it feels more like a test for `env`.

Reply:
Sure, I can remove one of them. I used the same approach here like in `uses cwd from options for each command`-test.
As far as I understand, there are not many more option to distinguish the same command here in the tests.

Reply:
Commited a simplified version of the test